README.md

```diff
@@ -66,15 +66,15 @@ IRIDA Next pipelines attempt to follow as much as possible the [nf-core][] guide
 * [Nf-core guidelines][nf-core-guidelines]
 * [Nf-core adding a new pipeline][nf-core-add-pipeline]
 
-While we do follow nf-core as closely as possible, there are a number of differences or additions we make for integrating pipelines into IRIDA Next. In particular, while pipelines can be contributed to the [nf-core index of pipelines][nf-core-pipelines], we do not require them to be as we may have different requirements for our own pipelines. Hence, some of the nf-core requirements may not apply if you do not intend to contribute the pipeline to nf-core.
+While we do follow nf-core as closely as possible, there are a number of differences or additions we make for integrating pipelines into IRIDA Next. Hence, some of the nf-core requirements may not apply if you do not intend to contribute the pipeline to nf-core.
 
 ## 1.1. Developing a pipeline to contribute to nf-core
 
 If you intend to develop a pipeline to contribute to nf-core, please follow all the [nf-core requirements][nf-core-requirements] in addition to the requirements listed in this guide.
 
 ## 1.2. Developing a pipeline which won't be contributed to nf-core
 
-If, instead, the pipeline won't be contributed to nf-core, then the only following set of [nf-core requirements][nf-core-requirements] would need to be followed in addition to the [if the guidelines don't fit][nf-core-external-development] section of the nf-core guidelines, [using nf-core components outside of nf-core][nf-core-outside-nf-core] section, and the below IRIDA Next requirements sections.
+If, instead, the pipeline won't be contributed to nf-core, then the only following set of nf-core requirements would need to be followed along with the below IRIDA Next requirements sections.
```

Comment:
If, instead, the pipeline won't be contributed to nf-core, then only the following set of nf-core requirements would need to be followed, along with the IRIDA Next requirements sections below.

Reply:
Thanks so much. Fixed in ee1165dcaf58a14ac12ec5cac35ef07f5ed7c374